modules/heat_conduction/src/materials/GapConductance.C

```diff
@@ -214,8 +214,20 @@ GapConductance::GapConductance(const InputParameters & parameters)
 void
 GapConductance::initialSetup()
 {
+  /// set generated from the passed in vector of subdomain names
+  const auto & check_subdomains = blockRestricted() ? blockIDs() : meshBlockIDs();
```

Comment:
@dschwen I checked one failed test, and it outputs blockRestricted() = 1, but blockIDs() is empty and meshBlockIDs() has two entities. Do you have any thoughts? One thing I am thinking is that the GapConductance seems to be a boundary material, and the boundary info seems to be reasonable. But I am not sure how to get coord_system from boundary information.